core/kazoo_number_manager/src/knm_numbers.erl

```diff
@@ -506,8 +506,11 @@ account_listing(AccountDb=?MATCH_ACCOUNT_ENCODED(_,_,_)) ->
             [{kz_doc:id(JObj), kz_json:get_value(<<"value">>, JObj)}
              || JObj <- JObjs
             ];
+        {'error', 'not_found'=_R} ->
+            lager:error("error listing numbers for ~s: ~p", [AccountDb, _R]),
+            [];
         {'error', _R} ->
-            lager:debug("error listing numbers for ~s: ~p", [AccountDb, _R])
+            lager:error("error listing numbers for ~s: ~p", [AccountDb, _R])
```

Comment:
I'm confused why you don't either throw an error or return an empty list too? It will crash further up (or the caller) but it would be nice to have it immediately crash here instead, no? Stacktrace should then point here as the cause.